driftdb-worker/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![doc = include_str!("../README.md")]
```

Comment:
What's this do?

Reply:
It imports the markdown as a docstring, so that it appears on docs.rs. See e.g. https://docs.rs/aper/latest/aper/ and have a look at [the source](https://docs.rs/aper/latest/src/aper/lib.rs.html)